lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -63,6 +63,7 @@
     font-size: $font-small;
     color: $dark-gray;
     font-style: italic;
+    height: 60px;
```

Comment:
Can you add a comment explaining that this is so all the cards have the same height?